sql-odbc/src/PowerBIConnector/OdfeSqlOdbcPBIConnector.pq

```diff
@@ -61,21 +56,9 @@ OdfeSqlOdbcPBIConnectorImpl = (Host as text, optional Port as number) as table =
                     UseSSL = 0
                 ],
 
-        // Set host & port in connection string. 
-        // Do not include port in connection string for aws server connection.
-        Server = 
-            if Port <> null then
-            [
-                Host = Host,
-                Port = Port
-            ]
-            else
-            [
-                Host = Host
-            ],
-
         ConnectionString = [
-            Driver = "ODFE SQL ODBC Driver"
+            Driver = "ODFE SQL ODBC Driver",
+            Host = Server
```

Comment:
Do we need to do anything on the driver side to handle this change?
(ie. will the driver understand a `Host` value of `localhost:9200`?)

Reply:
The driver doesn't need any change. I tested with PBI and ODBC administrator.